source/quick_start_guide/index.html.md.erb

```diff
@@ -26,8 +26,9 @@ If you want to build a technical integration between your service and the
 GOV.UK Pay API, your service team should have the necessary skills. You can
 refer to the [GOV.UK Service
 Manual](https://www.gov.uk/service-manual/the-team/what-each-role-does-in-service-team#roles-your-team-must-have)
-for more information. This does not apply if you only use [payment
-links](https://www.payments.service.gov.uk/govuk-payment-pages/).
+for more information.
+
+<%= warning_text('This does not apply if you only use <a href="https://www.payments.service.gov.uk/govuk-payment-pages/" target="_blank">payment links</a>.') %>
```

Comment:
some of this text does apply and some doesn't.

- all users should read about Pay and create an account (and probs read the service manual).
- only API users need to look at the swagger file

Maybe we introduce earlier the idea of the two types of integration, ie decide if you want to integrate with the API or use payment links. (instead of the warning, which maybe feels a but aggressive to the poor payment link users!)

Perhaps there's some content frm our new Get Started page that is relevant here?

Reply:
we could add in the great content from Justin - the stuff from the "Decide how you want to take payments" section of the get started content?